src/miner.cpp

```diff
@@ -395,10 +393,8 @@ bool static ScanHash(const CBlockHeader *pblock, uint32_t& nNonce, uint256 *phas
             return true;
 
         // If nothing found after trying for a while, return -1
-        if ((nNonce & 0xffff) == 0)
-            return false;
         if ((nNonce & 0xfff) == 0)
-            boost::this_thread::interruption_point();
+            return false;
```

Comment:
@maaku suggested on #4793 when I changed this ```boost::this_thread::interruption_point()``` to ```return false``` that the change belonged to the removal of the hashmeter as it was the reason for the interruption point to be there in the first place.